src/test/kotlin/com/github/suusan2go/kotlinfillclass/inspections/FillClassInspectionTest.kt

```diff
@@ -8,75 +8,90 @@ import org.jetbrains.kotlin.idea.KotlinFileType
 
 class FillClassInspectionTest : BasePlatformTestCase() {
     fun `test fill class constructor`() {
-        doAvailableTest("""
```

Comment:
Sorry for the reformat noise, but ktlint was complaining, so I ran a format with it.